### ISSUE TITLE
new 0.3 force close when opening in windows 10

### ISSUE THREAD

**Issue by tangsgod** [tranxuanthang/lrcget#47]
i completly uninstall and remove data from the previous version and restarted my pc

i use LRCGET_0.3.0_x64-setup.exe

it is installing but it doesn't launch properly, it forced close

**Comment by thangtranxuan-vti** [comment]
I cannot reproduce this even on newly installed Windows 10. You can try clearing the database of LRCGET to see if it works:

Paste the following path to File Explorer:

```
%APPDATA%\Roaming\net.lrclib.lrcget
```

Then delete all the files inside (`db.sqlite3`, `db.sqlite3-shm`, `db.sqlite3-wal`).

**Comment by tranxuanthang** [comment]
I cannot reproduce this even on a newly installed Windows 10 (virtual machine). You can try clearing the database of LRCGET to see if it works:

Paste the following path to File Explorer:

%APPDATA%\Roaming\net.lrclib.lrcget

Then delete all the files inside (db.sqlite3, db.sqlite3-shm, db.sqlite3-wal).

**Comment by tangsgod** [comment]
Mmmh, I just did as you said, but it is still crashing

See the video here:
[]([](https://streamable.com/ndk2wc))

**Comment by tranxuanthang** [comment]
Could you try to run this in the PowerShell (Windows button right click > Windows PowerShell)?

```
C:\"Program Files"\LRCGET\LRCGET.exe | echo
```

(assuming you installed LRCGET at `C:\Program Files` location).

It will allow you to see more verbose logs, so more information to guess what went wrong. It should look like the picture below:

![image](https://github.com/tranxuanthang/lrcget/assets/15942946/f0a623ae-79ce-4f87-99b2-e5c2b8a26c51)

You can check the installed location by right clicking to LRCGET shortcut > Properties > check the Target field:

![image](https://github.com/tranxuanthang/lrcget/assets/15942946/521dcd9c-76ca-4bde-a190-926ee07525ef)

**Comment by tangsgod** [comment]
Here is the default location for me:
C:\Users\Utilisateur\AppData\Local\LRCGET\LRCGET.exe

Here the results:

**Comment by tangsgod** [comment]
and it still doesn't work...

**Comment by tranxuanthang** [comment]
I expected some kind of panicked message in stdout/stderr but there is none. I still have no idea what went wrong here.

I will make a custom built version for you to debug this, but it'll take some time, maybe 1-2 days.

**Comment by tangsgod** [comment]
Could it be because the installation location is different from yours?

**Comment by tranxuanthang** [comment]
> Could it be because the installation location is different from yours?

I don't think so.

There are to kind of setup file for Windows:

-  LRCGET_0.3.0_x64-setup.exe
-  LRCGET_0.3.0_x64_en-US.msi 

One will install it in the `Program Files` folder, the other will install it at `%APPDATA%\Local`. I've tested and both should work the same.

**Comment by tangsgod** [comment]
I just tried the .msi and had the same result

I also deleted the .exe file, redownload it, reinstall it but no luck there too

What is bothering me is that each time I reinstall the 0.2.1 version it works

**Comment by tangsgod** [comment]
@tranxuanthang forget it, it works!

I don't know why it didn't before but now it is working as expected.

I'll test it against my music librairie (a few TB) a let you know how it is.

I'm sorry for the inconvenience, great work !

**Comment by tranxuanthang** [comment]
Though I curious what actually went wrong but I'm glad it works now!

**Comment by Hector47** [comment]
Hello, I could reproduce the same issue on my windows server (2019, based on windows 10):

When connecting remotely to it (using RDP via MRemoteNG), the default settings is to disable the sound. No audio device was available in Windows.
Maybe the rewrite in Rust dosen't handle this case.

As soon as I activated the audio playback (Bring to this computer), the programe started working.

Thanks for your help

**Comment by Sparkrdom** [comment]
I can confirm that it has something to do with the audio device. Enabling audio playback fixed it for me as well.

**Comment by tangsgod** [comment]
> I can confirm that it has something to do with the audio device. Enabling audio playback fixed it for me as well.

I'm still struggling, with this, how would you do it with the Anydsek software?

**Comment by tangsgod** [comment]
@tranxuanthang any fix for this coming soon?

**Comment by Sparkrdom** [comment]
> > I can confirm that it has something to do with the audio device. Enabling audio playback fixed it for me as well.
> 
> I'm still struggling, with this, how would you do it with the Anydsek software?

From what their knowledge base says, you can enable audio through any desk by going to Settings > Audio.
https://support.anydesk.com/knowledge/settings#audio

**Comment by guitarallstar** [comment]
Seems I share a similar predicament. This app crashes on startup on my Win10 machine. I have tried the 2 installers, different versions and install locations, updating & restarting PC. No version of LRCGET appears to work on my machine and the PowerShell command returns nothing.

**Comment by tranxuanthang** [comment]
@guitarallstar Did you run any script to debloat Windows or remove Microsoft Edge? Please check this issue https://github.com/tranxuanthang/lrcget/issues/45 to see if it works for you.

**Comment by guitarallstar** [comment]
> @guitarallstar Did you run any script to debloat Windows or remove Microsoft Edge? Please check this issue #45 to see if it works for you.

Yes sir I must have debloated. I apologize I did not see this other ticket with a solution, but that did indeed fix it after installing Edge and restarting PC.

**Comment by Flightkick** [comment]
I noticed the same behavior as OP when trying to open the application through a remote connection with Remmina.

Setting `Audio output mode` to `Local` adds an audio output device to the remote Windows 10 host and allows the application to start.

**Comment by tranxuanthang** [comment]
Allow me to close this issue to clean up, since this is related to a very very old version.
